spec/components/schemas/Scheduling/InvoiceDueTimeScheduleInstruction.yaml

```
@@ -0,0 +1,19 @@
+type: object
+description: |
+  The calculation instruction of invoice due time shift.
+  This is used in conjunction with the **billing anchor settings** to calculate the time the invoice gets paid.
```

```suggestion
  This is used in conjunction with the **billing anchor settings** to calculate the invoice due time.
```
It's not necessarily paid unless autopay is enabled.

---

right! thx